src/service-card.js

```diff
@@ -0,0 +1,71 @@
+import { LitElement, html, css } from 'lit-element/lit-element.js';
```

Comment:
I moved this into src because that's a common developer convention for SOURCE files